app/javascript/controllers/map_controller.js

```diff
@@ -106,4 +111,18 @@ export default class MapController extends Controller {
 
     return { discreteAir, linearAir };
   }
+
+  updateMap() {
+    const pollutant = document.querySelector("select#pollutant").value;
```

Comment:
I think you could use Stimulus here to 'target' this element, rather than vanilla JS: https://stimulus.hotwired.dev/handbook/hello-stimulus.

This is what we've done in the tab_controller (although I am in the process of stripping this out because we now need Turbo Streams to handle the additional complexity).

Reply:
Good shout, I've added a commit with this change for both this element and the map element.